contessa/models.py

```diff
@@ -36,6 +36,7 @@ class QualityCheck(AbstractConcreteBase, DQBase):
     id = Column(BIGINT, primary_key=True)
     attribute = Column(TEXT)
     rule_name = Column(TEXT)
+    rule_type = Column(TEXT)
```

Comment:
we need to provide some way of migrating to upper version for user.

it can be automatic, when contessa finds out there is mismatch between models. or we can provide cli that would allow user to easily do migration - with or without data migration according to usera needs

what do you think ?

Reply:
done in https://github.com/kiwicom/contessa/tree/cechovsky/migrations
already merged